### PR TITLE
Add support for npm priority queue 

### PR DESCRIPTION
**Pull request by keshav-space**

### CODE REVIEW

File: minecode/visitors/npm.py [hunk 1]
```diff
@@ -103,3 +106,73 @@ def get_uris(self, content):
...
+def map_npm_package(package_url):
+    """
+    Add a npm `package_url` to the PackageDB.
+
+    Return an error string if any errors are encountered during the process
+    """
+    from minecode.management.commands.priority_queue import add_package_to_scan_queue
+    from minecode.management.commands.run_map import merge_or_create_package
+
+    pacakge_json = get_package_json(
```
Comment (JonoYang) [inline minecode/visitors/npm.py:139]: There's a typo here and on line 145
```suggestion
    package_json = get_package_json(
```

Comment (JonoYang) [inline minecode/visitors/npm.py:139]: I think `get_package_json` should be wrapped in a try-except block in the case where the request fails, we can return the request error message.

**Review by JonoYang — CHANGES_REQUESTED** [review]
@keshav-space Thanks for the PR! I've left some comments.

File: minecode/visitors/npm.py (hunk 1 above)
Comment (keshav-space) [reply minecode/visitors/npm.py]: Already handling request error inside `get_package_json`.
Do we still want to wrap it inside try-except block?
https://github.com/nexB/purldb/blob/4ce38fb525de74891f27b4df21deb2ad27c20e73/minecode/visitors/npm.py#L121-L128

Comment (JonoYang) [reply minecode/visitors/npm.py]: I think you're good. I misred the code.

**Comment by JonoYang** [comment]
@keshav-space Changes look good, merging!

**Pull request merged**